lib/cdo/properties.rb

```diff
@@ -36,156 +36,157 @@ def self.delete(key)
   end
 
   def self.get_user_metrics
-    # Include stale default values as of 2016-06-22 so we never show 0. These
+    # Include stale default values as of 2016-11-23 so we never show 0. These
     # would be used, for example, if the DB is unavailable or the cron failed to
     # run properly.
     get(:about_stats) || {
       'percent_female' => 43,
-      'number_served' => 256_924_978,
-      'number_students' => 11_309_238,
-      'number_teachers' => 340_979
+      'number_served' => 296_598_685,
+      'number_students' => 14_126_655,
+      'number_teachers' => 451_385
     }
   end
 end
 
 def fetch_metrics
-  # Include stale default values as of 2016-01-04 so we never show 0. These
+  # Include stale default values as of 2016-11-23 so we never show 0. These
   # would be used, for example, if the DB is unavailable.
   Properties.get(:metrics) || {
-    'created_at' => "2016-01-04T21:37:19+00:00",
-    'created_on' => "2016-01-04",
-    'petition_signatures' => 2_053_571,
-    'lines_of_code' => 11_151_730_618,
+    'created_at' => '2016-11-23T19:48:27+00:00',
+    'created_on' => '2016-11-23',
+    'petition_signatures' => 1_712_470,
```

Comment:
Note this number has dropped as the result of us counting unique email addresses rather than counting email addresses (see #11932).